open_academy/i18n/es_MX.po

```diff
@@ -0,0 +1,206 @@
+msgid "Attendee"
+msgstr "Asistente"
+
+#. module: open_academy
+#: model:ir.model,name:open_academy.model_res_partner
+msgid "Contact"
+msgstr "Contacto"
+
+#. module: open_academy
+#: model:ir.model,name:open_academy.model_course
+#: model:ir.model.fields,field_description:open_academy.field_session__course_id
+msgid "Course"
+msgstr "Curso"
+
+#. module: open_academy
+#: model:ir.actions.act_window,name:open_academy.course_action
+#: model:ir.ui.menu,name:open_academy.course_menu
+#: model:ir.ui.menu,name:open_academy.open_academy_root_menu
+msgid "Courses"
+msgstr "Cursos"
+
+#. module: open_academy
+#: model:ir.model.fields,field_description:open_academy.field_attendee__create_uid
+#: model:ir.model.fields,field_description:open_academy.field_course__create_uid
+#: model:ir.model.fields,field_description:open_academy.field_session__create_uid
+msgid "Created by"
+msgstr "Creado por"
+
+#. module: open_academy
+#: model:ir.model.fields,field_description:open_academy.field_attendee__create_date
+#: model:ir.model.fields,field_description:open_academy.field_course__create_date
+#: model:ir.model.fields,field_description:open_academy.field_session__create_date
+msgid "Created on"
+msgstr "Creado en"
+
+#. module: open_academy
+#: model:ir.model.fields,field_description:open_academy.field_course__description
+#: model_terms:ir.ui.view,arch_db:open_academy.course_view_form
+msgid "Description"
+msgstr "Descripción"
+
+#. module: open_academy
+#: model_terms:ir.ui.view,arch_db:open_academy.attendee_view_form
+msgid "Discard"
+msgstr "Descartar"
+
+#. module: open_academy
+#: model:ir.model.fields,field_description:open_academy.field_attendee__display_name
+#: model:ir.model.fields,field_description:open_academy.field_course__display_name
+#: model:ir.model.fields,field_description:open_academy.field_session__display_name
+msgid "Display Name"
+msgstr "Nombre a Mostrar"
+
+#. module: open_academy
+#: model:ir.model.fields,field_description:open_academy.field_session__duration
+msgid "Duration"
+msgstr "Duración"
+
+#. module: open_academy
+#: model_terms:ir.ui.view,arch_db:open_academy.course_view_search
+msgid "Group By"
+msgstr "Agrupar Por"
+
+#. module: open_academy
+#: model:ir.model.fields,field_description:open_academy.field_attendee__id
+#: model:ir.model.fields,field_description:open_academy.field_course__id
+#: model:ir.model.fields,field_description:open_academy.field_session__id
+msgid "ID"
+msgstr "ID"
+
+#. module: open_academy
+#: model:ir.model.fields,field_description:open_academy.field_res_partner__instructor
+#: model:ir.model.fields,field_description:open_academy.field_res_users__instructor
+#: model:ir.model.fields,field_description:open_academy.field_session__instructor_id
+msgid "Instructor"
+msgstr "Instructor"
+
+#. module: open_academy
+#: code:addons/open_academy/models/session.py:0
+#, python-format
+msgid "Instructor not present in the attendees of his/her own session"
```

Comment:
Change this message to "Instructor must not be an attendee of his/her own session"